ipapython/ipaldap.py

```diff
@@ -1391,22 +1397,25 @@ def find_entries(self, filter=None, attrs_list=None, base_dn=None,
         if attrs_list:
             attrs_list = [a.lower() for a in set(attrs_list)]
 
-        sctrls = None
         cookie = ''
         page_size = (size_limit if size_limit > 0 else 2000) - 1
         if page_size == 0:
             paged_search = False
 
+        sctrls = []
+        if paged_search:
+            sctrls.append(SimplePagedResultsControl(0, page_size, cookie))
```

Comment:
That's also wrong and breaks paged control. :(

You have to create a new ``SimplePagedResultsControl`` with the cookie from the result control. With paged control, the server returns a resumption cookie in a result control for every query. The caller has to supply the resumption cookie in a following LDAP query to get the next page. The initial paged result request has an empty cookie.